jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/util/Constants.java

```diff
@@ -20,5 +20,6 @@
 public class Constants {
     public static final String RESOURCE_SOURCE_URL_ANNOTATION = "maven.jkube.io/source-url";
     public static final String RESOURCE_APP_CATALOG_ANNOTATION = "maven.jkube.io/app-catalog";
+    public static final String EXPOSE_LABEL = "expose";
```

Comment:
In general this is considered a bad idea and an anti-pattern (https://en.wikipedia.org/wiki/Constant_interface#:~:text=In%20general%2C%20collecting%20system%20constants,be%20considered%20an%20anti%2Dpattern.)

It's better to keep constants near their usage so that context doesn't get lost. Having an `EXPOSE_LABEL` constant with no other context makes it very hard to understand what its purpose is.